R/legendHatched.R

```diff
@@ -0,0 +1,309 @@
+#' @title Legend for Hatched Maps
+#' @description Plot legend for hatched maps.
+#' @name legendHatched
+#' @param pos position of the legend, one of "topleft", "top", 
+#' "topright", "right", "bottomright", "bottom", "bottomleft", 
+#' "bottomleftextra", "left" or a vector of two coordinates in map units 
+#' (c(x, y)). 
+#' @param title.txt title of the legend.
+#' @param title.cex size of the legend title.
+#' @param values.cex size of the values in the legend.
+#' @param categ vector of categories.
+#' @param patterns vector of patterns to be created for each element on \code{categ}, see \link{hatchedLayer}.
+#' @param ptrn.bg background of the legend box for each \code{categ}.
+#' @param ptrn.text text to be used for each \code{categ="text"}, as a single value or a vector.
+#' @param dot.cex cex of each \code{patterns = "dot"} categories, as a single value or a vector.
+#' @param text.cex text size of each \code{patterns = "text"} categories, as a single value or a vector.
+#' @param cex size of the legend. 2 means two times bigger.
+#' @param frame whether to add a frame to the legend (TRUE) or 
+#' not (FALSE).
+#' @param ... optional graphical parameters, see details on \link{hatchedLayer}
```

Comment:
After 179d12c9fa1ceebe8ac7505dc72285673edf030c it should work. Now working for all possibilities on `hatchedLayer` except `cellsize`:

- It is not purely a paramater for `hatchedLayer`
- I don`t think it may sense when plotting a legend to specify size on kms or meters

Let me know what you think, I could add a NOTE or WARNING

Reply:
I think it's ok like that for the moment. 
But I have the feeling that there is room for improvement in hatchedLayer and legendHatched. I'll probably get back to you later with some propositions....

Reply:
Happy to receive your feedback, I think this package is great, happy also to help